jsapp/js/components/map.es6

```diff
@@ -816,14 +816,14 @@ export class FormMap extends React.Component {
               })}
             </div>
             <div className='maplist-legend' onClick={this.toggleLegend}>
-              <i className={classNames('fa', this.state.showExpandedLegend ? 'fa-angle-down' : 'fa-angle-up')} /> {t('Legend')}
+              <i className={classNames('k-icon', this.state.showExpandedLegend ? 'k-icon-down' : 'k-icon-up')} /> {t('Legend')}
             </div>
           </bem.FormView__mapList>
         }
         {!this.state.markers && !this.state.heatmap &&
           <bem.Loading>
             <bem.Loading__inner>
-              <i />
+              <i className='k-spin k-icon k-icon-spinner'/>
```

Comment:
`<LoadingSpinner/>` and `<ui.LoadingSpinner/>` are the same thing. The difference is in how it was imported. Some places import whole `ui.es6` (for example because the need multiple things from it), and other just need this single thing.

As for places that uses the `<bem.Loading>` - the `LoadingSpinner` always displays the spinner icon plus text message. I didn't want to make it more configurable for now, as we would need the icon to be optional (and the `k-spin` class too), the header text (some have `<h3>` in there) and message text optional too.

Reply:
And this particular place doesn't have a message, and `LoadingSpinner` always displays a message